shared/actions/settings.js

```diff
@@ -53,42 +67,68 @@ function deleteAccountForever (): DeleteAccountForever {
   return {type: Constants.deleteAccountForever, payload: undefined}
 }
 
-function * saveNotificationsSaga (): SagaGenerator<any, any> {
+function * _onUpdatePGPSettings (): SagaGenerator<any, any> {
```

Comment:
i think related to me removing the try blocks for the TODO global handler stuff which is actually already in...